### PR TITLE
Provide import resolution linting

### PR DESCRIPTION
**Pull request by andy-hook**
- Provide import path error checking
- Improve dev experience in visual editors
- Clean up pre-existing lint errors and warnings

### CODE REVIEW

File: components/converter/ConverterContent.js [hunk 1]
```diff
@@ -1,57 +0,0 @@
-import React from 'react'
-import styled from 'styled-components'
-import { breakpoint } from 'lib/microsite-logic'
-import { useWeb3Connect } from 'lib/web3-connect'
-import Balance from './Balance'
-import Form from './Form'
-import Info from './Info'
-import Providers from './Providers'
```
Comment (andy-hook) [inline components/converter/ConverterContent.js:8]: Unused file as far as I could see, these paths were invalid so removed the whole thing.

File: jsconfig.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (sohkai) [inline jsconfig.json:1]: Just wondering on this; should this be something we start configuring in all our repos for VSCode users?

It's a bit annoying VSCode "leaks" this configuration.

Comment (andy-hook) [reply jsconfig.json:1]: I agree it’s not ideal but I think their intention is standardising javascript projects through config in a similar manner to the enhancements that `tsconfig` offers. I’d definitely be in favour of adopting in other projects, it’s a good stopgap for providing dev exp benefits without using strict typing in the short term.

Comment (sohkai) [reply jsconfig.json:1]: Let's do it then!

**Pull request merged**